resource-managers/yarn/src/test/scala/org/apache/spark/network/yarn/YarnShuffleServiceMetricsSuite.scala

```diff
@@ -37,28 +38,57 @@ class YarnShuffleServiceMetricsSuite extends SparkFunSuite with Matchers {
   val metrics = new ExternalBlockHandler(streamManager, blockResolver).getAllMetrics
 
   test("metrics named as expected") {
-    val allMetrics = Set(
+    val allMetrics = Seq(
```

Comment:
Why `Seq` here?

Reply:
I think the comment below explains. You can't sort a `Set` so if we use `Set` here we have to call `toSeq.sorted` -- we might as well just use `Seq` to begin with.

Reply:
This contribution is also orthogonal to the PR's main contribution but this looks okay in this case since it's a trivial.